test/Microsoft.Sbom.Targets.Tests/Utility/GeneratedSbomValidator.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Targets.Tests.Utility;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Sbom.Api.Utils;
+using Microsoft.Sbom.Contracts;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+/// <summary>
+/// This class is used to validate that the generated SBOM has valid fields and data.
+/// </summary>
+#pragma warning disable CA5350 // Suppress Do Not Use Weak Cryptographic Algorithms as we use SHA1 intentionally
+internal class GeneratedSbomValidator
+{
+    private readonly SbomSpecification sbomSpecification;
+
+    public GeneratedSbomValidator(SbomSpecification sbomSpecification)
+    {
+        this.sbomSpecification = sbomSpecification;
+    }
+
+    internal void AssertSbomIsValid(string manifestPath,
```

Comment:
nit: would put this on its own line

Reply:
Thanks. Addressed ce6a1dc8146dc3f2e630f1de71d79966634f4c97